src/modules/map/components/olMap/services/StyleService.js

```diff
@@ -1,15 +1,29 @@
 import { $injector } from '../../../../../injection';
-import { baseStyleFunction, measureStyleFunction } from '../olStyleUtils';
-
-
-
+import { markerStyleFunction, highlightStyleFunction, highlightTemporaryStyleFunction, measureStyleFunction, nullStyleFunction, lineStyleFunction, polygonStyleFunction, textStyleFunction, rgbToHex } from '../olStyleUtils';
 
 /**
  * @enum
  */
 export const StyleTypes = Object.freeze({
+	NULL: 'null',
 	MEASURE: 'measure',
-	DRAW: 'draw'
+	HIGHLIGHT: 'highlight',
+	HIGHLIGHT_TEMP: 'highlight_temp',
+	DRAW: 'draw',
+	MARKER: 'marker',
+	TEXT: 'text',
+	LINE: 'line',
+	POLYGON: 'polygon'
+});
+
+
+/**
+ * @enum
+ */
+export const StyleSizeTypes = Object.freeze({
+	SMALL: 'small',
+	MEDIUM: 'medium',
+	BIG: 'big'
```

Comment:
Should be moved to a new file called `styles.js` under `src/services/domain`, because we access it from our toolbox module.